tests/ui/perfherder/alerts-view/status_dropdown_test.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { render, waitFor, cleanup, fireEvent } from '@testing-library/react';
+
+import testAlertSummaries from '../../mock/alert_summaries';
+import testPerformanceTags from '../../mock/performance_tags';
+import repos from '../../mock/repositories';
+import StatusDropdown from '../../../../ui/perfherder/alerts/StatusDropdown';
+
+const testAlertSummary = testAlertSummaries[0];
+const testAlerts = testAlertSummary.alerts;
+const testRepoModel = repos[2];
+
+const testUser = {
+  username: 'mozilla-ldap/test_user@mozilla.com',
+  isLoggedIn: true,
+  isStaff: true,
+  email: 'test_user@mozilla.com',
+};
+
+const testIssueTrackers = [
+  {
+    id: 1,
+    text: 'Bugzilla',
+    issueTrackerUrl: 'https://bugzilla.mozilla.org/show_bug.cgi?id=',
+  },
+  {
+    id: 2,
+    text: 'Github - Servo',
+    issueTrackerUrl: 'https://github.com/servo/servo/pull/',
+  },
+];
+
+const testStatusDropdown = (summaryTags) => {
+  testAlertSummary.performance_tags = summaryTags;
+
+  return render(
+    <StatusDropdown
+      alertSummary={testAlertSummary}
+      user={testUser}
+      updateState={() => {}}
+      repoModel={testRepoModel}
+      updateViewState={() => {}}
+      issueTrackers={testIssueTrackers}
```

Comment:
We could simply reference the associated [Django fixture](https://github.com/mozilla/treeherder/blob/4cfdae3f41d656840486b771501f6498836e7710/treeherder/perf/fixtures/issue_tracker.json).